collectors/tc.plugin/README.md

```diff
@@ -166,16 +163,89 @@ And this is what you are going to get:
+tc filter add dev eth5 protocol ip parent 2: prio 1 u32 match ip dsfield 0x40 0xff classid 2:16
+tc filter add dev eth5 protocol ip parent 2: prio 1 u32 match ip dsfield 0x48 0xff classid 2:18
+tc filter add dev eth5 protocol ip parent 2: prio 1 u32 match ip dsfield 0x60 0xff classid 2:24
+tc filter add dev eth5 protocol ip parent 2: prio 1 u32 match ip dsfield 0x68 0xff classid 2:26
+tc filter add dev eth5 protocol ip parent 2: prio 1 u32 match ip dsfield 0x80 0xff classid 2:32
+tc filter add dev eth5 protocol ip parent 2: prio 1 u32 match ip dsfield 0x88 0xff classid 2:34
+tc filter add dev eth5 protocol ip parent 2: prio 1 u32 match ip dsfield 0xa0 0xff classid 2:40
+tc filter add dev eth5 protocol ip parent 2: prio 1 u32 match ip dsfield 0xb8 0xff classid 2:46
+tc filter add dev eth5 protocol ip parent 2: prio 1 u32 match ip dsfield 0xc0 0xff classid 2:48
+
+tc qdisc add dev eth6 root handle 1: netem loss 0%
+tc qdisc add dev eth6 parent 1: handle 2: htb default 1
+tc class add dev eth6 parent 2: classid 2:1 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:8 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:10 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:16 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:18 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:24 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:26 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:32 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:34 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:40 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:46 htb rate 1gbit burst 1gbit
+tc class add dev eth6 parent 2: classid 2:48 htb rate 1gbit burst 1gbit
+
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0x20 0xff classid 2:8
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0x28 0xff classid 2:10
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0x40 0xff classid 2:16
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0x48 0xff classid 2:18
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0x60 0xff classid 2:24
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0x68 0xff classid 2:26
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0x80 0xff classid 2:32
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0x88 0xff classid 2:34
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0xa0 0xff classid 2:40
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0xb8 0xff classid 2:46
+tc filter add dev eth6 protocol ip parent 2: prio 1 u32 match ip dsfield 0xc0 0xff classid 2:48
```

Comment:
I don't like this part. We are giving a specific QoS ruleset to users, about some `eth5` and `eth6` network interfaces, with static rates and bursts. Why do we need this?

I have written FireQoS to avoid exactly that.

Reply:
That's why it's the easy way and the hard way. Someone already added such instructions in the issue I mentioned in the documentation (#4563). I could show just ~5 lines instead of the whole thing. What do you think?

Reply:
I modified it, please check again.